apis/beacon/light_client/instant_update.yaml

```diff
@@ -0,0 +1,59 @@
+get:
+  operationId: getLightClientInstantUpdate
+  summary: Get a `LightClientInstantUpdate` for the given (recent) block header
+  description: |
+    Requests the best [`LightClientInstantUpdate`](../../../types/altair/light_client.yaml#/Altair/LightClientInstantUpdate) known by the server for the given block header.
+    Depending on the `Accept` header it can be returned either as JSON or SSZ-serialized bytes.
+
+    Servers providing this endpoint SHOULD always listen to the sync committee signature gossip and collect individual signatures. On request the best BLS
+    signature is aggregated and returned for the specified (recent) block header. When a new block appears, the previously known best sync aggregate for its
+    parent is compared against the sync aggregate found in the new block and replaced if the canonical one is better. Best sync aggregates are retained and
+    served for the 16 most recent slots.
+    Note that since always listening to the signature gossip costs some resources, it is acceptable to only start listening once this endpoint is called
+    and stop listening if it is not called for an extended period of time.
```

Comment:
So there's `SyncCommitteeMessage` and `SyncCommitteeContribution`.. can we line up what's being asked for here so that its better undesrtood?

It does concern me listening to all gossip on everything running the beacon-api, especially for home stakers etc it's overhead they shouldn't require.. 
Aggregates will be available before a block is produced anyway, I'm wondering why we'd be pushing to listen to individual messages?

Reply:
Actually we don't necessarily have to specify here how to obtain the best signatures as it does not affect the way the endpoint is used, from the consumer perspective it just affects the delays. Listening to all individual sync committee sigs is indeed expensive and it was my mistake to specify it as the default option. Listening to the four subnet aggregates is definitely better as a default suggestion. I changed the description accordingly. I see no reason to specifically forbid beacon nodes from listening to all sigs but it should not be a mandatory thing to implement (and even if implemented by a certain consensus client it should probably not be the default option). I'd even be fine with making this whole feature optional to implement, I just want to come up with a spec that makes sense so those who are interested in implementing it can stay compatible with each other.